source/versions/legacy-versions.rst

```diff
@@ -0,0 +1,51 @@
+    See :doc:`filenames` for information on newer versions.
+
+The format of the SpongeForge download filename is:
+``spongeforge-<MCVersion>-<ForgeBuildId>-<SpongeAPIVersion>-BETA-<SpongeBuildId>.jar``
+
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``MCVersion``        | The Minecraft version. Only clients compatible with this version can connect.                 |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``ForgeBuild``       | The Forge build that SpongeForge is built against and is guaranteed to be compatible          |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``SpongeAPIVersion`` | The version of SpongeAPI implemented by this file. This is what Sponge plugins depend on.     |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``SpongeBuildId``    | The build number of Sponge. This is what you should supply when reporting bugs or seeking     |
+|                      | support.                                                                                      |
++----------------------+-----------------------------------------------------------------------------------------------+
+
+The Forge Build in the filename specifies the version of Forge this version of SpongeForge used during development and 
+testing. The two versions are guaranteed to work without any issues. We **tend** to use the latest *Recommended Build* 
+of Forge for this purpose.
+
+.. note::
+
+    Normal Forge mods can usually run on any build of Forge for a given Minecraft version (e.g. 1.12.2) without any 
+    problem. However, SpongeForge needs to access, among other things, internal parts of Forge, which most mods
+    shouldn’t be touching, let alone modifying as Sponge does. Since Forge is free to change their internal code
+    whenever they want to, its normal guarantee of backwards-compatibility doesn’t apply to SpongeForge. Feel free to
+    use more recent versions of Forge than the one used for SpongeForge, but we can't always guarantee compatibility.
+
+.. warning::
+
+    When investigating crash issues, you can freely try newer versions of Forge than listed on the SpongeForge Jar.
+    However, it is recommended to also check with the matching version, to make sure your issue is not related to a
+    version mismatch.
+    Even though there will be no guarantee of compatibility, please report any breakage to the issue tracker, so that
+    SpongeForge can be updated.
+
+Example
+-------
+
+SpongeForge Jar files will always follow this naming scheme, to allow you to easily identify compatibility.
+
+For example, the file name ``spongeforge-1.12.2-2705-7.1.0-BETA-3442.jar`` is compatible with Minecraft version
```

Comment:
See above

Reply:
This filename was released (see [SpongeForge download](https://www.spongepowered.org/downloads/spongeforge/stable/1.12.2?until=2018-09-14T15%3A28%3A16Z)), so I left it.